src/controllers/project.controller/updateProject/index.js

```diff
@@ -1,7 +1,8 @@
 const { validationResult } = require('express-validator');
 const catchAsync = require('../../../utils/catchAsync');
 const { query: Hasura } = require('../../../utils/hasura');
-const { updatePost_query } = require('./queries/queries');
+const { updatePost_query, updateRolesRequired, updateProjectFlags, updateDocuments, updateProjectTags, updateMentions, deleteTeam } = require('./queries/mutations');
```

Comment:
Please use strict camel case for all variable names updatePostQuery or updatePostMutation